### PR TITLE
Add ereolen-homepage-url prop to all dpl-react apps

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-160
https://reload.atlassian.net/browse/DDFLSBP-199

#### Description
Adding url data prop for ereol-homepage for [this PR](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/654).

#### Screenshot of the result
n/a
#### Additional comments or questions
n/a

### CODE REVIEW

File: web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php [hunk 1]
```diff
@@ -156,7 +157,14 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
...
       '#description' => $this->t('My page in ereolen', [], ['context' => 'Library Agency Configuration']),
-      '#default_value' => $config->get('ereolen_my_page_url'),
+      '#default_value' => $config->get('ereolen_my_page_url') ?? self::EREOLEN_MY_PAGE_URL,
+    ];
+
+    $form['reservations']['ereolen_homepage_url'] = [
+      '#type' => 'url',
+      '#title' => $this->t('Ereolen home link', [], ['context' => 'Library Agency Configuration']),
+      '#description' => $this->t('Home page in ereolen', [], ['context' => 'Library Agency Configuration']),
+      '#default_value' => $config->get('ereolen_homepage_url') ?? self::EREOLEN_HOMEPAGE_URL,
```
Comment (kasperg) [inline web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php:167]: Please consider the configuration here.

I see that it is the responsibility of the CMS to inject these urls but:

1. Is it necessary that they are configurable? I am not sure that Ereolen changes url structures very often.
2. If they are to be configurable I suggest that we provide a sensible default for these values based on the current version of the site.

File: web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php [hunk 2]
```diff
@@ -156,7 +157,14 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#type' => 'url',
       '#title' => $this->t('Ereolen link', [], ['context' => 'Library Agency Configuration']),
       '#description' => $this->t('My page in ereolen', [], ['context' => 'Library Agency Configuration']),
-      '#default_value' => $config->get('ereolen_my_page_url'),
+      '#default_value' => $config->get('ereolen_my_page_url') ?? self::EREOLEN_MY_PAGE_URL,
+    ];
```
Comment (kasperg) [inline web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php:161]: :+1: Nice of you to remember this default.

**Review by kasperg — CHANGES_REQUESTED** [review]
:+1: Nice work adding this to the CMS. I have a quibble regarding how much we expect libraries to configure on their own for the site to work.

File: web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php (hunk 1 above)
Comment (Dresse) [reply web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php:167]: I also have a small comment, but this is regarding the translations in:
web/profiles/dpl_cms/translations/da.po

Should the new strings not be added and translated here?

**Review by kasperg — APPROVED** [review]
:+1. LGTM

**Pull request merged**